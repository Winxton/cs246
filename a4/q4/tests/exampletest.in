w_Hello_World.txt translate e 3 swapcase doublevowels doublevowels
w_Hello_World.txt skipword translate o 0

