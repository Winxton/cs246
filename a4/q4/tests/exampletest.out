h3LLOOOO
wOOOORLD
W0rld
